backend/main.py

```diff
@@ -72,30 +69,12 @@ def preprocessing():
     Preprocess data for all strategy and save results locally,
```

Comment:
Can we add comments to explain request args, including mandatory and optional ones

Reply:
This HTTP endpoint would be deprecated very soon as multi-level downsampling is coming up, more args are required for new downsample method. 
This function will be temporarily removed.